src/stores/routePathStore.ts

```diff
@@ -20,16 +19,25 @@ export enum RoutePathViewTab {
     List,
 }
 
+export interface UndoObject {
+    routePathLinks: IRoutePathLink[];
+}
+
 export class RoutePathStore {
     @observable private _routePath: IRoutePath|null;
     @observable private _oldRoutePath: IRoutePath|null;
     @observable private _isGeometryValid: boolean;
     @observable private _neighborRoutePathLinks: IRoutePathLink[];
+    // TODO: Move out of store to state, rename as highlightedMapObject
     @observable private _highlightedObject: string | null;
+    // TODO: Move out of store to state, rename as extendedListItems:
```

Comment:
Regarding comments, I agree that we could change this. but be consistent in the naming highlightedObjects and extendedObjects points to the same set of items, so we should use consistent naming. Not highlightedMapObjects and extendedListItems => items and objects.

Reply:
:+1: , changed highlightedMapObjects to highlightedMapItems. It was a bad name anyway because the type was string, not object